main.nf

```diff
@@ -158,6 +158,7 @@ if(genome_file ){
 }
 
 process fastp {
+	publishDir "/${OUTDIR}/versions", mode: 'copy' , overwrite: 'true', pattern: '*versions.yml'
```

Comment:
Why is the `OUTDIR` prepended with "/"? If it is a "nextflow root" folder without the root slash, why not add the "/" to the variable itself?

Reply:
extra precaution, never hurts to have an extra slash

Reply:
I see! What about this then:

```
OUTDIR = '/' + params.outdir+'/'+params.subdir
```